alerting/src/test/kotlin/org/opensearch/alerting/TestHelpers.kt

```diff
@@ -213,6 +213,10 @@ val LEGACY_OPENDISTRO_DESTINATION_BASE_URI = "/_opendistro/_alerting/destination
 val ALWAYS_RUN = Script("return true")
 val NEVER_RUN = Script("return false")
 val DRYRUN_MONITOR = mapOf("dryrun" to "true")
+val TEST_INDEX_1 = "hr_data"
+val TEST_INDEX_2 = "not_hr_data"
+val TEST_ROLE = "hr_role"
+val TEST_BACKEND_ROLE = "HR"
```

Comment:
Add more description to the name like `TEST_HR_INDEX`, `TEST_NON_HR_INDEX`, `TEST_HR_ROLE`, `TEST_HR_BACKEND_ROLE`. For the non-hr index, this can be named as a different kind of index like `TEST_IT_INDEX`.

Reply:
Done